pandas/core/arrays/base.py

```diff
@@ -461,6 +461,8 @@ def shift(self, periods=1):
             The number of periods to shift. Negative values are allowed
             for shifting backwards.
 
+        .. versionadded:: 0.24.0
+            fill_value : None or value, default None (NaN)
```

Comment:
The `.. versionadded`  directive goes after the description

    fill_value : object, optional
        The scalar value to use for newly introduced missing values.
        The default is ``self.dtype.na_value``.
    
        .. versionadded:: 0.24.0

Reply:
changed